torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -100,14 +101,24 @@ def special_pattern_replacement(model: QuantizedGraphModule) -> QuantizedGraphMo
     nodes = list(model.graph.nodes)
     for n in model.graph.nodes:
         q_node = n
-        if q_node.target == torch.quantize_per_tensor:
+        if q_node.target == torch.quantize_per_tensor or \
+           (q_node.op == "call_method" and q_node.target == "to" and q_node.args[1] == torch.float16):
+            ref_node = q_node.args[0]
             # get output scale/zero_point/dtype from the quantize node
-            ref_node, scale_node, zero_point_node, dtype = q_node.args
+            # ref_node, scale_node, zero_point_node, dtype = q_node.args
+            # TODO: add safety checks that users for the ref_node and dq_node needs to be one
 
             is_call_function, is_call_method, is_call_module = check_node(ref_node, modules)
             if is_call_module or is_call_function or is_call_method:
                 dq_node = ref_node.args[0]
-                if dq_node.target == 'dequantize':
+                assert isinstance(dq_node, Node) or isinstance(dq_node, (tuple, list))
+                if isinstance(dq_node, Node):
+                    is_dequantize = dq_node.target == 'dequantize'
+                elif isinstance(dq_node, (tuple, list)):
+                    is_dequantize = all(x.target == 'dequantize' for x in dq_node)
+                else:
+                    is_dequantize = False
```

Comment:
Do we ever hit this case since `dq_node` must be Node or tuple or list (assert in L114)?

Reply:
yeah.. we won't hit this, I can change to initialize is_dquantize to False I think, in case we missed it in the branches in the future